ansible/tasks/setup-system.yml

```diff
@@ -37,6 +37,14 @@
     update_cache: yes
     cache_valid_time: 3600
 
+- name: Install other useful tools
+  become: yes
+  apt:
+    pkg:
+      - magic-wormhole
```

Comment:
What are you planning to use this for Div?

Reply:
Transferring files/logs as needed